### ISSUE TITLE
Accessible goal chart (numbers and strings)

### ISSUE THREAD

**Issue by myasonik** [elastic/elastic-charts#1160]
Goals have few data points: 

- min
- max
- target
- value
- label major
- label minor

min, max, target, and value we should incorporate into the `<dl>`

The major label should be incorporated into the accessible labels for the chart
